packages/ui/src/views/repo/components/index.ts

```diff
@@ -1,2 +1,3 @@
 export * from './branch-selector/branch-selector'
+export * from './branch-selector/branch-selector-dropdown'
```

Comment:
We shouldn't export the `./branch-selector/branch-selector-dropdown` component, as it is essentially a part of `./branch-selector/branch-selector`. If we need the types, let's create a `types.ts` file in `views/repo/components/branch-selector` and move all the types there. Additionally, we can create an `index.ts` file in the same folder to export the types and the branch-selector component.
This architecture has been approved by Kevin.

Reply:
sure, sounds good